core.py

```diff
@@ -98,46 +129,70 @@ def enable_motors(self, enable):
         if self.arduino:
             self.arduino.enable_motors(enable)
 
-    def throttle(self, left_speed, right_speed):
+    def throttle(self,
+                 left_speed,
+                 right_speed,
+                 left_servo=ServoEnum.LEFT_MOTOR_ESC,
+                 right_servo=ServoEnum.RIGHT_MOTOR_ESC):
         """ Send motors speed value in range [-1,1]
             where 0 = neutral """
 
         # Calculate microseconds from command speed
-        left_micros = self.left_servo.micros(left_speed)
-        right_micros = self.right_servo.micros(right_speed)
+        left_micros = 0
+        right_micros = 0
+        try:
+            if left_servo != ServoEnum.SERVO_NONE:
+                left_micros = self.servos[left_servo][0].micros(left_speed)
+            if right_servo != ServoEnum.SERVO_NONE:
+                right_micros = self.servos[right_servo][0].micros(right_speed)
+        except:
```

Comment:
Is a wide open `except` here what you want? They're usually a warning flag that you'll end up catching exceptions that perhaps you may actually want to raise higher in the stack. What failure mode are you expecting?